src/Sulu/Bundle/ContentBundle/Resources/views/Template/content-types/smart_content.html.twig

```diff
@@ -11,13 +11,15 @@
      data-aura-locale="{{ languageCode }}"
      data-aura-has='{{ params.has | json_encode }}'
      data-aura-display-options='{{ sulu_parameter_to_key_value(params.display_options.value) | json_encode }}'
-     {% if params.has.datasource %}
+{% if params.has.datasource %}
      data-aura-datasource='{{ params.datasource | json_encode }}'
-     {% endif %}
+{% endif %}
      {% if params.has.categories %}
      data-aura-category-root='{{ params.category_root.value }}'
-     {% endif %}
+{% endif %}
```

Comment:
because twig ignores such lines. if you indent the spaces will be in the output also.